### PR TITLE
Hide sensitive info while displaying configuration

### PR DESCRIPTION
**Pull request by PrasadG193**

### CODE REVIEW

**Comment by PrasadG193** [comment]
image

File: pkg/execute/executor.go [hunk 1]
```diff
@@ -234,19 +235,34 @@ func runPingCommand(args []string, clusterName string) string {
...
 	b, err := ioutil.ReadAll(file)
 	if err != nil {
-		return "", err
+		return configYaml, err
 	}
 
-	return string(b), nil
+	c := &config.Config{}
+	if len(b) != 0 {
+		yaml.Unmarshal(b, c)
```
Comment (ssudake21) [inline pkg/execute/executor.go]: do we expect any errors in unmarshal ?

File: pkg/execute/executor.go [hunk 2]
```diff
@@ -234,19 +235,34 @@ func runPingCommand(args []string, clusterName string) string {
...
+	}
+
+	// hide sensitive info
+	c.Communications.Slack.Token = ""
+	c.Communications.ElasticSearch.Password = ""
```
Comment (ssudake21) [inline pkg/execute/executor.go]: if we have omitempty above why do we need to set empty strings again ?

Comment (PrasadG193) [reply pkg/execute/executor.go]: `omitempty` drops fields if they are empty while Marshaling. When we Unmarshal yaml to struct, we get Token and Password set in the struct. So in order to drop the fields, we need to make their value empty

File: pkg/execute/executor.go (hunk 1 above)
Comment (PrasadG193) [reply pkg/execute/executor.go]: In the worst case, we will get empty values of keys don't match. Will handle error though

**Pull request merged**